docs/wiki/[Examples]-Create-and-Assign-Custom-RBAC-Roles.md

```diff
@@ -0,0 +1,128 @@
+## Overview
+
+This page describes how to use the module to create your own custom RBAC roles and assign them at the appropriate scopes within your Enterprise-Scale deployment.
+
+In this example, we will create a custom RBAC role called "Reader Support Tickets" which will allow members to read everything in a subscription and to also open support tickets.
+
+We will then assign the new RBAC Role at the `es_landingzones` scope.
+
+We will update the built-in configuration with the new custom RBAC role by following these steps:
+
+- Create the role definition file
+
+- Save the new role at the `es_root` scope
+
+- Assign the role at the `es_landingzones` scope
+
+The files we will use are:
+
+- [lib/role_definitions/role_definition_es_reader_support_tickets.tmpl.json](#librole_definitionsrole_definition_es_reader_support_ticketstmpljson)
+- [lib/archetype_extension_es_root.tmpl.json](#libarchetype_extension_es_roottmpljson)
+- [lib/archetype_extension_es_landing_zones.tmpl.json](#libarchetype_extension_es_landing_zones_tmpljson)
+
+>IMPORTANT: To allow the declaration of custom or expanded templates, you must create a custom library folder within the root module and include the path to this folder using the `library_path` variable within the module configuration. In our example, the directory is `/lib`.
+
+## Create Role Definition File
+
+In your `/lib` directory create a `role_definitions` subdirectory if you don't already have one. You can learn more about archetypes and custom libraries in [this article](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/wiki/%5BUser-Guide%5D-Archetype-Definitions).
+
+>NOTE: Creating a `role_definitions` subdirectory is a recommendation only. If you prefer not to create one or to call it something else, the custom roles will still work.
+
+In the `role_definitions` subdirectory, create a `role_definition_es_reader_support_tickets.tmpl.json` file. This file will contain the role definition for our `Reader Support Tickets` role.
+
+To attempt to ensure that every custom role we create has a unique value for "name", we use the uuidv5 function that is built-in to Terraform. You can learn more about uuidv5 [here](https://www.terraform.io/language/functions/uuidv5).
```

Comment:
Did we consider using Powershell `New-Guid` or `uuidgen` on Linux or macOS?

As long as the UUID is unique it doesn't matter, the `uuidv5()` function is useful when you want a consistent UUID, but in this case it really doesn't matter.

Use of Terraform console requires the working directory to have `terraform init` run, thus creating a barrier.

Reply:
Updated to include `New-Guid` and `uuidgen` as options.